locales/es_MX/LC_MESSAGES/duckduckgo.po

```diff
@@ -614,7 +631,7 @@ msgid "Canada"
 msgstr "Canadá"
 
 msgid "Canada (fr)"
-msgstr ""
+msgstr "Canadá (Fr)"
```

Comment:
I understand your point in having the language codes capitalized, but as I said, it's better to keep things consistent with the original strings.

```suggestion
msgstr "Canadá (fr)"
```